common/types/admin.go

```diff
@@ -277,6 +278,12 @@ func (v *MembershipInfo) GetRings() (o []*RingInfo) {
 	return
 }
 
+// PersistenceInfo is an internal type which exposes persistence configuration
+type PersistenceInfo struct {
+	Name                      *string `json:"name,omitempty"`
```

Comment:
Looks like the `Name` field is only for visibility persistence? I assume we will also add info for the persistence for execution and history to the describe cluster result, so may be we can rename this to something like VisibilityPersistenceName?

Reply:
err, yes. Good catch!